src/test/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStepWorkflowTest.java

```diff
@@ -282,4 +283,24 @@ public void sshAgentDocker() throws Exception {
         });
     }
 
+    @Issue("JENKINS-32104")
+    @Test
+    public void testMissingCredential() {
```

Comment:
More extra credit if you are interested: test coverage for `ignoreMissing`, which seems to have been forgotten in #4 along with the incorrect handling of the default case.